### DISCUSSION TITLE
Freezing Block Move Test on SuperMicro X10DRH-i

### DISCUSSION THREAD

**Discussion by laozhoubuluo** [memtest86plus/memtest86plus#211] [Q&A] [answered]
I built a new computer on November 14th and it passed the memory test that day using Memtest86+ v6.00. Considering various reasons, I upgraded its BIOS and IPMI on November 15th, but I found that the system cannot pass Block Move Test at November 16th, the system will freeze at 7%, no matter running the test only or from first test onwards trying 3 times will always get stuck there.

![](https://user-images.githubusercontent.com/31075951/202247986-27fe812f-de03-41cb-a6d3-3cf6a1fbbaca.png)

I tried swapping the order of the two memories but it didn't work. When I thought it was a memory or other parts failure, I tested the device again with Memtest86 v4.3.7 with Block Move Test, and the failure no longer occurred in this 2 times.

![](https://user-images.githubusercontent.com/31075951/202247467-b2581f73-d16e-41f6-96f4-03e43ae24ced.png)

I can't determine what causes this problem, whether it is a memory module failure or a UEFI BIOS version problem, or something else. I hope someone can help, and I am very grateful.

The system configuration is:
CPU: Single Intel Xeon E5-2696v4
Motherboard: Supermicro X10DRH-i
Memory: Samsung DDR4 ECC 2400MHz (M386A4G40EM2-CRC) 32G*2
BIOS Version: 3.4a (08/16/2021)
IPMI Version: 03.94 (10/01/2021)
Factory BIOS Version: 3.1a (04/09/2019)
Factory IPMI Version: 03.80 (02/14/2019)

BIOS upgrade logs:
![](https://user-images.githubusercontent.com/31075951/202247265-511ccd4e-01b4-49f0-89fb-b4168dc4000f.png)

**Comment by yoshinoyatoko** [comment, marked as answer]
Could you try to force one cpu core to run block move test? 
not use all cores to run memtest

**Reply by laozhoubuluo** [reply]
The following results are from the latest Memtest86+ v6.10.
When disabling SMP (now disabling by default), it ran twice without any error.
![](https://user-images.githubusercontent.com/31075951/217513487-34358ef1-1398-4577-852c-5bc936d7d85e.png)
Due to #244, it is currently not possible to manually enable SMP, so it is not possible to confirm if SMP is causing the problem.
![](https://user-images.githubusercontent.com/31075951/217516771-bbe6fd8e-6d0c-4e7e-bb7e-02455bc0202f.png)
Considering this question has been properly circumvented, I'll add a circumvention hint at #233 and close this discussion.
